frontend/www/js/omegaup/components/problem/FilterDifficulty.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="card mt-4">
+    <div class="card-body">
+      <h3>{{ T.wordsDifficulty }}</h3>
+      <div
+        v-for="(difficulty, index) in difficulties"
+        :key="index"
+        class="form-check"
+      >
+        <label class="form-check-label">
+          <input
+            class="form-check-input"
+            type="radio"
+            name="difficulty"
+            :value="difficulty"
+            @change="$emit('change', $event.target.value)"
+          />{{ T[difficulty] }}
+        </label>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Model } from 'vue-property-decorator';
+import T from '../../lang';
+
+@Component
+export default class FilterDifficulty extends Vue {
+  @Model('change', { type: String }) readonly selectedDifficulty:
+    | null
+    | string = null;
+
+  T = T;
+  difficulties: string[] = [
+    'qualityFormDifficultyEasy',
+    'qualityFormDifficultyMedium',
+    'qualityFormDifficultyHard',
+  ];
```

Comment:
aquí lo más conveniente es usar:

```suggestion
  difficulties: { [key: string]: string; } = {
    qualityFormDifficultyEasy: T.qualityFormDifficultyEasy,
    qualityFormDifficultyMedium: T.qualityFormDifficultyMedium,
    qualityFormDifficultyHard: T.qualityFormDifficultyHard,
  };
```

la razón es que queremos evitar el uso de `T[...]` y en vez siempre usar `T.miCadena` para poder detectar los casos en los que una cadena no está siendo usada correctamente.